src/qt/utilitydialog.h

```diff
@@ -22,7 +22,7 @@ class HelpMessageDialog : public QDialog
     Q_OBJECT
 
 public:
-    explicit HelpMessageDialog(QWidget *parent, bool about);
+    explicit HelpMessageDialog(QWidget *parent, bool about, bool wallet_enabled = true);
```

Comment:
Not sure if we need `= true` default. `HelpMessageDialog` is not something that is used a lot (only twice?), and if it was, assuming `= true` as default would probably be error prone, as it has to actually depend on the build.

Also, `bitcoingui.h` has `bool enableWallet = false;` defined, so at least it's inconsistency.

Reply:
btw, I think that combining two independent dialogs in one widget is a bad implementation.

Reply:
[Updated](https://github.com/bitcoin-core/gui/pull/227#issuecomment-786746698).